README.md

```diff
@@ -348,78 +348,78 @@ all:
 ```yaml
 - hosts: node1 node2
   vars:
-    ha_cluster_cluster_name: "my-new-cluster"
-    ha_cluster_hacluster_password: "password"
+    ha_cluster_cluster_name: my-new-cluster
+    ha_cluster_hacluster_password: password
     ha_cluster_resource_primitives:
-      - id: "xvm-fencing"
-        agent: "stonith:fence_xvm"
+      - id: xvm-fencing
+        agent: 'stonith:fence_xvm'
         instance_attrs:
           - attrs:
-              - name: "pcmk_host_list"
-                value: "node1 node2"
-      - id: "simple-resource"
-        agent: "ocf:pacemaker:Dummy"
-      - id: "resource-with-options"
-        agent: "ocf:pacemaker:Dummy"
+              - name: pcmk_host_list
+                value: node1 node2
+      - id: simple-resource
+        agent: 'ocf:pacemaker:Dummy'
+      - id: resource-with-options
+        agent: 'ocf:pacemaker:Dummy'
         instance_attrs:
           - attrs:
-              - name: "fake"
-                value: "fake-value"
-              - name: "passwd"
-                value: "passwd-value"
+              - name: fake
+                value: fake-value
+              - name: passwd
+                value: passwd-value
         meta_attrs:
           - attrs:
-              - name: "target-role"
-                value: "Started"
-              - name: "is-managed"
-                value: "true"
+              - name: target-role
+                value: Started
+              - name: is-managed
+                value: 'true'
         operations:
-          - action: "start"
+          - action: start
             attrs:
-              - name: "timeout"
-                value: "30"
-          - action: "monitor"
+              - name: timeout
+                value: 30
```

Comment:
This changes the "type" from string to integer - where are these values used?  If the only place they are used is with `{{ value }}` then that should be ok - that will convert to string

Reply:
Yes, they are only used in `{{ value }}`. In general, they are all strings, though. So I fixed that and added an example and a test for setting timeout and intervals with time units.